.spelling

```diff
@@ -81,6 +84,7 @@ Gmail
 Grafana
 Graphviz
 Hystrix
+httpbin.example.com
```

Comment:
This shouldn't be in the dictionary, use backticks on this word instead.

Reply:
Sure.